src/DynamoCore/Engine/CodeCompletion/CodeCompletionServices.cs

```diff
@@ -183,6 +183,7 @@ internal IEnumerable<CompletionData> SearchTypes(string stringToComplete, Elemen
                     candidates = type.GetOverloadsOnInstance(functionName);
                 }
             }
+
```

Comment:
Let's revert changes to any files that are only whitespace or unnecessary.

Reply:
will do